rahuldkjain/github-profile-readme-generator
# Resolved- Add Skills for Devops Engineer #747

## What type of PR is this? (check all applicable)


## Description
This PR addresses Issue #747, which involves adding major skills including Terraform, Ansible, Gitlab, Prometheus, Jira, Chef, and Kibana for the role of a DevOps Engineer.

## Related Tickets & Documents
- Issue #747
## QA Instructions
Please follow these instructions to test the changes:
1. Clone this branch and deploy it to a DevOps environment.
2. Verify that the major skills mentioned in the issue (Terraform, Ansible, Gitlab, Prometheus, Jira, Chef, Kibana) have been correctly added to the DevOps Engineer role and also added the respected SVG icon links to access publicly for each.
3. Ensure that these changes has been updated to `skills.js`
4. Perform any necessary tests to verify the functionality and correctness of the changes.

## Screenshots
**Before** 
![Screenshot 2023-10-04 004557](https://github.com/rahuldkjain/github-profile-readme-generator/assets/106306211/245a2f80-064a-4faf-bcf2-f622f2c9f56c)

![Screenshot 2023-10-04 004616](https://github.com/rahuldkjain/github-profile-readme-generator/assets/106306211/a7df3449-f717-4082-aa4f-10fbe1f8ed60)

**After**
DevOps - Terraform, Ansible, Gitlab, Prometheus, and Chef

![Screenshot 2023-10-04 003846](https://github.com/rahuldkjain/github-profile-readme-generator/assets/106306211/0d457ba7-41a4-4778-a8d9-c75411a2fb7e)


Other - Jira 

![Screenshot 2023-10-04 003909](https://github.com/rahuldkjain/github-profile-readme-generator/assets/106306211/517432e4-57b9-4c0b-b1f0-50279d5f02f6)


## Recordings

https://github.com/rahuldkjain/github-profile-readme-generator/assets/106306211/6b0b0f80-442e-48b0-8918-86c707bc8acf

## Added to documentation?
